homeassistant/components/azuredns/manifest.json

```diff
@@ -0,0 +1,14 @@
+{
+  "domain": "azuredns",
+  "name": "Azure DNS",
+  "documentation": "https://www.home-assistant.io/components/azuredns",
+  "requirements": [
+    "adal==1.2.1",
+    "azure-mgmt-dns==2.1.0",
+    "ip-query==4.3.0"
```

Comment:
Absolutely not. This is 10MB dependency. 

All other DNS integrations work because they are a dynamic DNS service and thus will do the translation from IP to DNS record. This integration does too many things. We should not merge this.

Reply:
Yes, in my previous PR I was doing an aiohttp request to `https://api.ipify.org` which is significantly smaller.